torch/_dynamo/variables/user_defined.py

```diff
@@ -563,3 +563,22 @@ def odict_getitem(self, tx, key):
         )(
             collections.OrderedDict.__getitem__(self.value, key.as_python_constant())
         ).add_options(key, self)
+
+
+class KeyedJaggedTensorVariable(UserDefinedObjectVariable):
+    @staticmethod
+    def is_matching_object(obj):
+        try:
+            from torchrec.sparse.jagged_tensor import KeyedJaggedTensor
```

Comment:
By the way, this is how we're doing it for a bunch of other out-of-torch matchers but this is bad, we really should have some other strategy for matching in this situation.